packages/web/src/scss/components/Modal/_ModalDialog.scss

```diff
@@ -1,24 +1,61 @@
 // 1. Intentionally transition also the worse performing properties to smooth out the changes of viewport size around
-//    the tablet breakpoint.
+//    the tablet breakpoint. However, `top` and `bottom` are not transitioned when switching from and to `auto`.
```

Comment:
> the tablet breakpoint

Does this apply only to the docked variant? Because I don't know what could cause troubles with the uniform view.

Reply:
Actually the alignment options cause the trouble:

- transitions between top ↔️ center and center ↔️ bottom are OK
- transitions between top ↔️ bottom is not possible